pkg/serverless/invocationlifecycle/invocation_processor.go

```diff
@@ -5,10 +5,12 @@
 
 package invocationlifecycle
 
+import serverlessLog "github.com/DataDog/datadog-agent/pkg/serverless/logs"
```

Comment:
nit : if `logs` does not conflict with any other packages, we don't need to rename it, so you could leave it as logs if you want